src/common/intl.cpp

```diff
@@ -167,7 +159,7 @@ const char* wxLanguageInfo::TrySetLocale() const
 
 const char* wxLanguageInfo::TrySetLocale() const
 {
-    return wxSetlocale(LC_ALL, CanonicalName);
+    return wxSetlocale(LC_ALL, (CanonicalRef.empty()) ? CanonicalName : CanonicalRef);
```

Comment:
Very minor, but why extra parenthesis?
```suggestion
    return wxSetlocale(LC_ALL, CanonicalRef.empty() ? CanonicalName : CanonicalRef);
```

Reply:
> Very minor, but why extra parenthesis?

Well, just a habit to include the condition of the conditional operator in parentheses. For a single term they are superfluous, of course.